srcpkgs/mobile-config-firefox/template

```diff
@@ -1,11 +1,11 @@
 # Template file for 'mobile-config-firefox'
 pkgname=mobile-config-firefox
-version=3.1.0
+version=3.2.0
 revision=1
 build_style=gnu-makefile
 short_desc="Mobile and privacy friendly configuration for Firefox"
 maintainer="Jami Kettunen <jami.kettunen@protonmail.com>"
 license="MPL-2.0"
 homepage="https://gitlab.com/postmarketOS/mobile-config-firefox"
 distfiles="${homepage}/-/archive/${version}/${pkgname}-${version}.tar.gz"
```

Comment:
```suggestion
distfiles="https://gitlab.com/postmarketOS/mobile-config-firefox/-/archive/${version}/mobile-config-firefox-${version}.tar.gz"
```

Reply:
Fair enough

Reply:
Made it an even shorter form because that one also works the same.